src/wcm.cpp

```diff
@@ -292,16 +292,21 @@ OptionWidget::OptionWidget(Option *option) : Gtk::Box(Gtk::ORIENTATION_HORIZONTA
         } else
         {
             auto combo_box = std::make_unique<Gtk::ComboBoxText>();
-            for (const auto & [name, _] : option->int_labels)
+            for (const auto & pair : option->int_labels)
```

Comment:
It's possible to use `for (const auto & [name, _] : option->int_labels) {...}` for `std::vector` of `std::pair`, please use it for more readability